README.md

```diff
@@ -137,6 +146,62 @@ your node further see [Optional configurations](#optional-configurations).
 * **IMAGE_TAG__[...]** - Use custom docker image for specified components.
 * **PORT__[...]** - Use custom port for specified components.
 
+## Obtaining a migrated L1 datadir
+
+For some node configurations a migrated L1 datadir is required, you can obtain
+one by following one of the options outlined below.
+
+### 1. Download a pre-migrated datadir
+
+If you do not have an existing L1 datadir but wish to full sync and/or run an
+archive node you can download a migrated datadir hosted by cLabs from one of the links below.
+
+* [Alfajores migrated datadir](https://storage.googleapis.com/cel2-rollup-files/alfajores/alfajores-migrated-datadir.tar.zst)
+* Baklava migrated datadir - pending network launch
+* Mainnet migrated datadir - pending network launch
+
+### 2. Migrate your own datadir
+
+If you have been running an existing L1 node and wish to continue using the same datadir,
+you can migrate the data in order to use it with the L2 node.
+
+Once an L2 hardfork date has been decided for a specific network the L1
+[blockchain client](https://github.com/celo-org/celo-blockchain) will be
+relesed with a hardcoded stop block. Nodes running this version will stop at
+the stop block, at which point the node can be stopped and the datadir can be
```

Comment:
"will stop at the stop block, at which point the node can be stopped" sounds irritating. It can be stopped after it stopped?
```suggestion
released with a hardcoded stop block. Nodes running this version will stop producing blocks at
the stop block, at which point the node can be shut down and the datadir can be
```